modules/claims_api/lib/claims_api/local_config.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'yaml'
+require 'erb'
+
+module ClaimsApi
+  # Simple configuration loader for Claims API module
+  module LocalConfig
```

Comment:
We can handle getting configuration parameters in the fes_service/base.rb class, rather than in a separate class.